doc/man3/MD5.pod

```diff
@@ -49,7 +49,13 @@ L<openssl_user_macros(7)>:
 
 All of the functions described on this page are deprecated.
 Applications should instead use L<EVP_DigestInit_ex(3)>, L<EVP_DigestUpdate(3)>
-and L<EVP_DigestFinal_ex(3)>.
+and L<EVP_DigestFinal_ex(3)>, or L<EVP_Digest(3)>.
+MD2(), MD4(), and MD5()
+can continue to be used by defining B<OPENSSL_COMPENSATE_DEPRECATION>.
+They can also be replaced by, e.g.,
+
+    EVP_Digest(data, count, md_buf, NULL, EVP_md5(), NULL)
```

Comment:
This should really be a fetch instead of EVP_md5()
EVP_Digest_ex() could pass a libctx, name and propq?

Reply:
Yeah, this is meanwhile taken over by `EVP_Q_digest()`.